fenix/app/src/main/java/org/mozilla/fenix/compose/Image.kt

```diff
@@ -35,6 +35,8 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * bounds defined by the width and height.
  * @param contentScale Optional scale parameter used to determine the aspect ratio scaling to be used
  * if the bounds are a different size from the intrinsic size of the [Painter].
+ * @param placeholder Optional placeholder image used when loading or an error occurs.
+ * By default set to a solid color in [DefaultImagePlaceholder].
```

Comment:
```suggestion
 * @param placeholder composable displayed when loading.
 * By default set to a solid color in [DefaultImagePlaceholder].
```

I think if we should have a separate param for `fallback`, the recommendations usecase uses the same composable in both cases but the API should be more clear with good default params IMO

Reply:
I am blind.. Added fallback param 👍